Documentation/templates/colors/literals-swift3.md

```diff
@@ -30,11 +30,11 @@ Note: if you use `enumName: UIColor` (or `NSColor` on macOS) then the color cons
 **Extract:**
 
 ```swift
-extension ColorName {
+internal extension ColorName {
   /// 0x339666ff (r: 51, g: 150, b: 102, a: 255)
-  static let articleBody = #colorLiteral(red: 0.2, green: 0.588235, blue: 0.4, alpha: 1.0)
+  internal static let articleBody = #colorLiteral(red: 0.2, green: 0.588235, blue: 0.4, alpha: 1.0)
```

Comment:
2) Or is it this one being the bad one, like since it's an `internal extension` that `internal` is redundant here?

I don't remember which are the exact rules but since I remember you fixed it in a recent PR when testing with the Swift 5 compiler warning about redundant access modifiers… better at least make all templates, and this all examples in the doc, consistent with each other

Reply:
Uhm, this is swift 3, which will never have that warning about redundant ACL modifiers. I'm not even sure what's valid in swift 3, and we haven't changed the templates for it in the swift 5 PR. I think we better keep this one as is (with the extra `internal`).